pkg/protocol/sybilprotection/sybilprotectionv1/sybilprotection.go

```diff
@@ -108,8 +98,42 @@ func (o *SybilProtection) Shutdown() {
 	o.TriggerStopped()
 }
 
-func (o *SybilProtection) TrackValidationBlock(block *blocks.Block) {
-	o.performanceTracker.TrackValidationBlock(block)
+func (o *SybilProtection) TrackBlock(block *blocks.Block) {
+	accountData, exists, err := o.ledger.Account(block.ProtocolBlock().IssuerID, block.SlotCommitmentID().Slot())
```

Comment:
why fetch the accountData before even checking if it is a candidacy announcement? Check first if we need to do the work or not.